### PR TITLE
Removed compression Plugin

### PR DESCRIPTION
**Pull request by SteffiAntony**
DEFLATE Data Compression Algorithm Used for HTTPS  which may be vulnerable to breach attack
Defect: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=579318 
Signed-off-by: Steffi Antony <“steffiantony196@gmail.com”>

### CODE REVIEW

File: src/views/Operations/FactoryReset/FactoryReset.vue [hunk 1]
```diff
@@ -98,6 +98,7 @@ export default {
   },
   methods: {
     onResetSubmit() {
+      console.log('Modal:-> ', this.$bvModal);
```
Comment (Nikhil-Ashoka) [inline src/views/Operations/FactoryReset/FactoryReset.vue]: Please remove console log

File: vue.config.js [hunk 2]
```diff
@@ -85,14 +83,6 @@ module.exports = {
...
       }
     }
-
-    if (process.env.NODE_ENV === 'production') {
-      config.plugins.push(
-        new CompressionPlugin({
-          deleteOriginalAssets: true,
-        })
-      );
-    }
```
Comment (joseph-reynolds) [inline vue.config.js:95]: I don't know the web application, so I cannot comment on this technical change.  It seems okay if the web application does not need to consume compressed content.

More important, I do not believe this chnage will fix the problem with the network scanner detecting a possible DEFLATE issue.  The network scanner scans the BMC's HTTPS interface, so if the BMC offers a compressed file -- regardless of if the web application uses it or not -- then the scanner will detect this issue.

Comment (Nikhil-Ashoka) [reply vue.config.js:95]: Resolved in discussion: https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=579318 - Comment 26

**Comment by Nikhil-Ashoka** [comment]
@joseph-reynolds, The impacted releases are 1030 and 1050. We can backport this change to them as well if needed. Your suggestion?

**Review by nabilpandiyalayil — APPROVED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]

**Pull request merged**